proto/babylonchain/babylon/v1beta1/scheduler.proto

```diff
@@ -1,23 +1,20 @@
 syntax = "proto3";
-package osmosis.meshsecurity.v1beta1;
+package babylonchain.babylon.v1beta1;
 
 // import "cosmos/base/v1beta1/coin.proto";
 import "gogoproto/gogo.proto";
 // import "amino/amino.proto";
 import "cosmos_proto/cosmos.proto";
 
-option go_package = "github.com/osmosis-labs/mesh-security-sdk/x/meshsecurity/types";
+option go_package = "github.com/babylonchain/babylon-sdk/x/babylon/types";
 option (gogoproto.goproto_getters_all) = false;
 option (gogoproto.equal_all) = false;
 
-//
-message ScheduledWork {
-  bool repeat = 1;
-}
-
+// ScheduledWork is XX
+message ScheduledWork { bool repeat = 1; }
```

Comment:
Here it is. Seems like a place-holder for future work. Perhaps we can remove in a follow-up.

Reply:
yep these will be replaced by our own APIs